src/planning/tests/planner-test.ts

```diff
@@ -375,26 +377,33 @@ ${particlesSpec}
 ${recipeManifest}
     `));
 
+    let key: (input: string) => (string | StorageKey);
```

Comment:
can you use storageKeyPrefixForTest (in handle-for-test.ts)? or modify it to take a string as input (this looks like it will be useful for other tests)

Reply:
storageKeyPrefixForTest makes volatile storage keys to use inside arcs; this is setting up manifest stores, which can't be volatile.